app/models/katello/concerns/pulp_content_unit.rb

```diff
@@ -0,0 +1,59 @@
+require 'set'
+
+module Katello
+  module Concerns::PulpContentUnit
```

Comment:
Should this be moved into app/services as well? Seems very Pulp layer specific and only being used by them.